src/subplots.ts

```diff
@@ -269,85 +265,118 @@ export class PlotPieChart extends PlotData {
-      this.refresh_buttons_coords();
+  private drawButtons(): void {
+    this.refresh_buttons_coords();
+    //Drawing the zooming button
+    Buttons.zoom_button(this.button_x, this.zoom_rect_y, this.button_w, this.button_h, this);
 
-      //Drawing the zooming button
-      Buttons.zoom_button(this.button_x, this.zoom_rect_y, this.button_w, this.button_h, this);
+    //Drawing the button for zooming window selection
+    Buttons.zoom_window_button(this.button_x,this.zw_y,this.button_w,this.button_h, this);
 
-      //Drawing the button for zooming window selection
-      Buttons.zoom_window_button(this.button_x,this.zw_y,this.button_w,this.button_h, this);
+    //Drawing the reset button
+    Buttons.reset_button(this.button_x, this.reset_rect_y, this.button_w, this.button_h, this);
 
-      //Drawing the reset button
-      Buttons.reset_button(this.button_x, this.reset_rect_y, this.button_w, this.button_h, this);
+    //Drawing the selection button
+    Buttons.selection_button(this.button_x, this.select_y, this.button_w, this.button_h, this);
 
-      //Drawing the selection button
-      Buttons.selection_button(this.button_x, this.select_y, this.button_w, this.button_h, this);
+    //Drawing the enable/disable graph button
+    Buttons.graph_buttons(this.graph1_button_y, this.graph1_button_w, this.graph1_button_h, '10px Arial', this);
 
-      //Drawing the enable/disable graph button
-      Buttons.graph_buttons(this.graph1_button_y, this.graph1_button_w, this.graph1_button_h, '10px Arial', this);
+    //draw permanent window button
+    Buttons.perm_window_button(this.button_x, this.perm_button_y, this.button_w, this.button_h, '10px Arial', this);
 
-      //draw permanent window button
-      Buttons.perm_window_button(this.button_x, this.perm_button_y, this.button_w, this.button_h, '10px Arial', this);
+    //draw clear point button
+    Buttons.clear_point_button(this.button_x, this.clear_point_button_y, this.button_w, this.button_h, '10px Arial', this);
 
-      //draw clear point button
-      Buttons.clear_point_button(this.button_x, this.clear_point_button_y, this.button_w, this.button_h, '10px Arial', this);
+    // Draw log scale buttons
+    Buttons.log_scale_buttons(this.button_x, this.xlog_button_y, this.ylog_button_y, this.button_w, this.button_h,
+      "10px Arial", this);
+    
+    // Draw Heatmap button
+    Buttons.heatmap_button(this.button_x, this.heatmap_button_y, this.button_w, this.button_h, "10px Arial", this);
+  }
 
-      // Draw log scale buttons
-      Buttons.log_scale_buttons(this.button_x, this.xlog_button_y, this.ylog_button_y, this.button_w, this.button_h,
-        "10px Arial", this);
-      
-      // Draw Heatmap button
-      Buttons.heatmap_button(this.button_x, this.heatmap_button_y, this.button_w, this.button_h, "10px Arial", this);
-    }
-    if (this.multiplot_manipulation) {
-      this.draw_manipulable_rect();
-    }
-    this.context.restore();
+  draw_from_context(hidden: any) {
```

Comment:
Could you add a comment to explain why this method is not doing anything please ? 
I think that this remainder will be useful in a (near?) future

Reply:
My bad, I was not explicit enough. 
I was thinking of an "in the code comment". 
But this is not really important as the vscode explanation is clear enough